src/plugins/data/common/search/aggs/metrics/cumulative_sum.ts

```diff
@@ -26,7 +26,7 @@ import { METRIC_TYPES } from './metric_agg_types';
 import { AggConfigSerialized, BaseAggParams } from '../types';
 
 export interface AggParamsCumulativeSum extends BaseAggParams {
-  buckets_path: string;
+  buckets_path?: string;
```

Comment:
Making buckets_path required in the expression functions blew up a bunch of functional tests. It should be optional for pipeline aggs.